scripts/upgrade_beeper_room.py

```diff
@@ -162,16 +177,12 @@ def make_url(part: str):
     "replacement_room": new_room_id,
 }
 
-# Tombstone the old room and make it so that you can't invite or send events to it
-# anymore.
-requests.put(
-    make_url(f"/rooms/{old_room_id}/state/m.room.tombstone"),
-    headers={"Authorization": f"Bearer {auth_token}"},
-    json=tombstone_content,
-)
-
+# Adjust the old room power levels first so Brad and Eric are allowed to tombstone the
+# room.
 old_power_level_content["events_default"] = 50
 old_power_level_content["invite"] = 50
+old_power_level_content["users"]["@eric:beeper.com"] = 100
+old_power_level_content["users"]["@brad:beeper.com"] = 100
```

Comment:
I don't think that this will work. You can't raise your own power level if you aren't powerful enough yet.

Reply:
Afaik this works because I'm a synapse admin. Other users tokens won't work for doing this.